src/test/java/posters/tests/smoke/RegisterFromUserMenuTest.java

```diff
@@ -1,49 +1,53 @@
-/**
- * 
- */
 package posters.tests.smoke;
 
 import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
+import com.xceptance.neodymium.util.DataUtils;
+
 import io.qameta.allure.Owner;
 import io.qameta.allure.Severity;
 import io.qameta.allure.SeverityLevel;
 import io.qameta.allure.junit4.Tag;
 import posters.dataobjects.User;
 import posters.flows.DeleteUserFlow;
 import posters.flows.OpenHomePageFlow;
-import posters.pageobjects.pages.browsing.HomePage;
-import posters.pageobjects.pages.user.LoginPage;
-import posters.pageobjects.pages.user.RegisterPage;
 import posters.tests.AbstractTest;
 
 /**
  * @author pfotenhauer
  */
+
 @Owner("Lisa Smith")
 @Severity(SeverityLevel.NORMAL)
 @Tag("smoke")
 @Tag("registered")
 public class RegisterFromUserMenuTest extends AbstractTest
 {
-    final User user = new User("Jane", "Doe", "jane@doe.com", "topsecret");
+    User user;
+
+    @Before
+    public void setup()
+    {
+        user = DataUtils.get(User.class);
+    }
 
     @Test
     public void testRegisteringFromUserMenu()
     {
         // Go to homepage
-        HomePage homePage = OpenHomePageFlow.flow();
+        var homePage = OpenHomePageFlow.flow();
```

Comment:
if we start using final for page objects we need to be consistent. But in general I would not add final to page objects since they are stateless and adding final doesn't provide advantages.

BTW: for test data objects adding final hints that they should not be changed

Reply:
Please adjust all test classes accordingly